style.css

```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+  height: 100vh;
+}
+
+.white-background {
+  background-color: white;
+  border-bottom-left-radius: 100px;
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 40px 24px 14px;
+}
+
+.logo {
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.menu img {
+  width: 24px;
+  height: 24px;
+}
+
+main {
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-size: 100% 90%;
+  background-repeat: no-repeat;
+  padding: 118px 24px;
+}
+
+.introduction {
```

Comment:
[OPTIONAL] Great job with the correct font use. But let's follow the guideline and fix the color of our font here and h1 as well. Kindly check the differences below.
<img width="385" alt="Screenshot 2023-02-24 at 9 43 53 AM" src="https://user-images.githubusercontent.com/25887253/221111146-e964a6e5-35aa-4f63-ae76-757196110961.png">
<img width="788" alt="Screenshot 2023-02-24 at 9 44 44 AM" src="https://user-images.githubusercontent.com/25887253/221111154-65ad9992-8978-4d62-ae7e-1cccb695e3b2.png">
<img width="385" alt="Screenshot 2023-02-24 at 9 43 23 AM" src="https://user-images.githubusercontent.com/25887253/221111140-0174455c-ff8c-4939-b54f-10da1dccaa66.png">

<img width="788" alt="Screenshot 2023-02-24 at 9 44 23 AM" src="https://user-images.githubusercontent.com/25887253/221111150-624a2d98-0f3b-405a-ae6c-81046b2ece7e.png">